scripts/setup/configure-rabbitmq

```diff
@@ -19,10 +19,14 @@ RABBITMQ_PASSWORD=$("$(dirname "$0")/../get-django-setting" RABBITMQ_PASSWORD)
 
 # Wait for RabbitMQ to start up
 retries=29
-while ! "${sudo[@]}" rabbitmqctl ping -q 2>/dev/null; do
+# Could use rabbitmqctl await_startup when we upgrade to 3.7.11.
+while ! "${sudo[@]}" rabbitmqctl -q await_online_nodes 1 2>/dev/null; do
+    if ((retries == 29)); then
+        echo "Waiting for RabbitMQ to start up..."
+    fi
     sleep 1
     if ! ((retries -= 1)); then
-        "${sudo[@]}" rabbitmqctl ping -q
+        "${sudo[@]}" rabbitmqctl -q await_online_nodes 1
```

Comment:
Should this also get a `2>/dev/null`?

Reply:
This is the final retry, so if it fails, we want to show the full error message and exit due to `set -e`.

Reply:
Aha, got it.